### PR TITLE
hover states

### PR DESCRIPTION
**Pull request by huwshimi**
Add hover states to added services buttons.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- hover the Added Services header, the arrow icon on the right hand side should change to a darker arrow
- click Added Services
- hover the header again and the close icon on the right hand side should change to a darker x

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2362/
Test FAILed.

**Comment by makyo** [comment]
NB this failure looks like an npm issue, will re-run

**Comment by makyo** [comment]
:+1: QA okay - hover states look great.  Hopefully CI just had a fit.

File: lib/views/browser/added-services-button.less [hunk 1]
```diff
@@ -10,6 +10,18 @@
     position: absolute;
     right: 20px;
   }
+
+  &:hover .action-indicator {
+    .normal {
```
Comment (hatched) [inline lib/views/browser/added-services-button.less]: inconsistent indentations here.

**Comment by hatched** [comment]
:+1: !

File: lib/views/browser/added-services-button.less (hunk 1 above)
Comment (huwshimi) [inline lib/views/browser/added-services-button.less]: Thanks, fixed!

**Comment by huwshimi** [comment]
Thanks for the reviews! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2367/
Test PASSed.

**Pull request merged**